### PR TITLE
Basegl npm

### PR DESCRIPTION
**Pull request by piotrMocz**
This changes the name of the package to `luna-basegl-ui` -- I really need this on master.

### CODE REVIEW

**Comment by wdanilo** [comment]
I've got request here. Can we not use the name `luna-basegl-ui` and instead use just `luna-ui`? Using `basegl` in the name is imo confusing and we will not be providing any other uis, so let keep it simple and obvious for users and contributors.

**Comment by piotrMocz** [comment]
Sure thing ;)

**Comment by wdanilo** [comment]
Awesome, thanks @piotrMocz . Btw we should also rename github repo `luna/basegl-ui` to `luna/gui`

File: update_statics.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 # echo "Installing the package..."
-# npm install
-# echo "Done."
+npm install
+echo "Done."
```
Comment (ljkania) [inline update_statics.sh]: We should have one consistent message. Please refactor to variable and use it along the project.

File: src/main.coffee [hunk 2]
```diff
@@ -16,7 +16,12 @@ import * as layers          from 'view/layers'
 import * as test            from './test'
 
 removeChildren = (name) =>
-    element = document.getElementById(name).innerHTML = ''
+    element = document.getElementById(name)
+    unless element?
+        console.error "The DOM element \"#{name}\" does not exist."
```
Comment (ljkania) [inline src/main.coffee]: Can we make this error string a function that takes `name` and returns error.

**Pull request merged**